README.md

```diff
@@ -121,6 +121,46 @@ render json: @posts, each_serializer: PostPreviewSerializer
 render json: @posts, serializer: CollectionSerializer, each_serializer: PostPreviewSerializer
 ```
 
+### Serialization Scope
+
+The [`serialization_scope` method](https://github.com/rails-api/active_model_serializers/blob/d02cd30fe55a3ea85e1d351b6e039620903c1871/lib/action_controller/serialization.rb#L13-L20)
+ allows you to include a method from the controller scope.
+By default, it is `:current_user`. For a richer example, let's imagine we want to use the controller
+method `current_admin` in our serializer, like so:
+
+```ruby
+class AdminUserSerializer < ActiveModel::Serializer
+  attributes :admin?
+  def admin?
+    current_admin.admin
+  end
+end
+```
+
+In order to do this, all we need to do is declare the `serialization_scope`:
+
+```ruby
+class SomeController < ActionController::Base
+  serialization_scope :current_admin
+
+  def current_admin
+    User.new(id: 2, name: 'Bob', admin: true)
+  end
+
+  def index
+    render json: User.new(id: 1, name: 'Pete', admin: false), serializer: AdminUserSerializer, adapter: :json_api
+  end
+end
```

Comment:
@bf4: Could you fill me in on what exactly we are testing here? It feels like we're making sure we can override the scope?